sys/sys/elf_common.h

```diff
@@ -382,6 +383,24 @@ typedef struct {
 #define	EF_RISCV_RVE		0x00000008
 #define	EF_RISCV_TSO		0x00000010
 
+/* Loongson LoongArch Specific e_flags
```

Comment:
Per https://man.freebsd.org/cgi/man.cgi?style(9) the comment should start on the second line, so:

```
/*
 * Loongson ...
 * ...
 */